enterprise/admin/views.py

```diff
@@ -532,11 +525,17 @@ def notify_enrolled_learners(cls, enterprise_customer, request, course_id, users
 
         with mail.get_connection() as email_conn:
             for user in users:
+                login_or_register = 'register' if isinstance(user, PendingEnterpriseCustomerUser) else 'login'
+                destination_url = logistration_url.format(
+                    site=enterprise_customer.site,
```

Comment:
This is looking good. I think you still want to use `enterprise_customer.site.siteconfiguration` to get the `LMS_ROOT_URL` instead of hard-coding `http` above.

Reply:
Also, you only need to format the `login_or_register` value inside the loop here, the rest could be formatted outside the loop.